fluent-package/apt/systemd-test/update-from-v4.sh

```diff
@@ -18,6 +19,9 @@ for d in $(seq 1 10); do
     touch /var/log/td-agent/$d.log
 done
 
+# When it updates from v4, the service needs to be stopped to restart fluentd after installation.
+sudo systemctl stop td-agent
```

Comment:
I see! Looks good.
Could you please leave a comment that this behavior is unexpected for us, such as the following?

```suggestion
# When it updates from v4, the service needs to be stopped to restart fluentd after installation.
# Note: This behavior is not expected, and maybe it should be fixed.
# The cause is unknown. For deb, usually, restart should happen when the service was active before the update.
sudo systemctl stop td-agent
```